src/Row.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export type Props = {
+  children: (React.ReactElement | React.ReactElement[])[];
+};
+
+export default "tr";
```

Comment:
Such a weird export both in the original file and here.

Reply:
I honestly don't understand this.